skopt/optimizer/base.py

```diff
@@ -127,13 +129,26 @@ def base_minimize(func, dimensions, base_estimator,
         exploration over exploitation and vice versa.
         Used when the acquisition is `"LCB"`.
 
-    * `n_jobs` [int, default=1]
+    * `n_jobs` [int, default=1]:
         Number of cores to run in parallel while running the lbfgs
         optimizations over the acquisition function. Valid only when
         `acq_optimizer` is set to "lbfgs."
         Defaults to 1 core. If `n_jobs=-1`, then number of jobs is set
         to number of cores.
 
+    * `include_time` [bool, default=False]:
+        If true, then `log(t(x))` is modelled along with `f(x)` where `t(x)`
+        is the time taken for the function call at x.
+        Note that
+
+            - `log(t) ~ N(m, sigma)`
+            - `-log(t) ~ N(-m, sigma)`
+            - `log(1/t)` ~ N(-m, sigma)
+
+        This implies `1/t` follows a lognormal distribution with mean `-m`
+        and `sigma` or `E(1/t) = exp(-m + sigma**2)`
+        The next point suggested is the point that minimizes `acq(x)*E(-1/t(x))`
```

Comment:
Just curious, do you have a reference for this strategy? 

A smart strategy, but which would require quite some changes in our codebase, is freeze-thaw bayesian optimization (http://arxiv.org/abs/1406.3896) where the idea is basically to extrapolate the function wrt some its parameters (like n_epochs). This is in turn allows to do early stopping, thereby reducing the total execution time.